repo/packages/D/dcos-ui/4/package.json

```
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "4.0",
+  "name": "dcos-ui",
+  "version": "4.0.1",
+  "tags": ["dcos-ui", "dcos", "ui"],
+  "maintainer": "frontend-dev@mesosphere.io",
```

Is this mail still working? Should we change it to `d2iq`? Who is in that mailing list? Now Mesosphere team is small maybe we should unify external mailing lists e.g.: `devs@d2iq.com` so we will not miss any email from our customers.

```suggestion
  "maintainer": "frontend-dev@d2iq.com",
```

---

☝️ @DominikDary is there such thing?

---

let's update that later on.